vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
The whole vendor folder should be gitignored.

Reply:
Done

Reply:
Done